autoload/resize_mode.vim

```diff
@@ -34,9 +34,9 @@ function! s:exit_mode(c, match_id) abort
   " Clear our highlighting
   call matchdelete(a:match_id)
 
-  " Not sure why, but couldn't get calling `echo` directly here to
-  " update the command line working. Do it later with feedkeys.
-  call feedkeys(":echo\<CR>", 'nt')
+  " Clear the -- RESIZE -- message
+  echo ''
+  redraw
```

Comment:
Not sure why this didn't work for you before, but the direct way works fine for me now in vim 9.1 (and this fixes an issue that the current implementation causes in terminal mode)

Reply:
If I were the author or maintainer, I would find this change a lot easier to review if your cover letter (PR description) or commit message explained what problem exists in terminal buffers and how these changes resolve it.

Reply:
The main problem is the absence of mappings for terminal mode, i.e. the plugin simply does noting in terminal mode.

The secondary problem (once the mappings for terminal mode are created), is that the above code for clearing the `-- RESIZE --` status message dumps the text `:echo` into the terminal buffer instead of doing what it was intending to do.

Hope this helps clarify the issue ❤️